### DISCUSSION TITLE
5 luglio 2022 - esercizio 1

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#14] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/ec73f043-40db-4f26-b5ec-9f6f25ab9d37)

**Comment by niccolo-fato** [comment]
Non sono sicuro sia corretto 
![IMAGE 2024-01-14 14:41:52](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/5691673b-746d-4bb2-8dab-c838f23f452e)

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/128373356/aae46403-33bd-47e6-ad68-076ad8988113)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/128373356/709e05d8-8c03-4a67-96cc-135734012cd0)

**Reply by LeoCosta02** [reply]
@niccolo-fato ho fatto qualche passaggio in più ma dovrebbe essere simile al tuo

**Reply by LeoCosta02** [reply]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/128373356/257db245-a062-4538-a94c-777ae3c789d3)

Il punto 1 giusto

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/c93e6e72-98b4-4a8a-8d7d-9678eb5ab49c)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/0bb9efb5-1ad4-4d95-814b-37dcf27d1a33)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/a8f1fc7e-a7ca-44db-9af4-efac3082c7fb)

**Comment by Elia-Belli** [comment]
**(1)**
- Giocatori che hanno segnato più di 1 goal: $Giocatori=\sigma_{(n-goal\gt 1)}Marcature$
- Giocatori che hanno segnato più di 1 goal in casa: $Giocatori\\_ Casa=\pi_{(\\#C,\\#S)}(\sigma_{(squadra-casa\ =\ \\#S)}((Giocatori \bowtie Gioca)\bowtie Partita))$
- Risposta: $\pi_{(\\#C,Nome,Cognome,Nsquadra)}((Calciatore \bowtie Giocatori\\_ Casa )\bowtie (\rho_{(Nsquadra\leftarrow Nome)}Squadra))$

**(2)**
- Calciatori Milan: $Calc\\_ MI=\pi_{(\\#C)}(((\rho_{(Nsquadra \leftarrow Nome)}(\sigma_{(Nome=Milan)}Squadra))\bowtie Gioca) \bowtie Calciatore)$
- Calciatori Milan che hanno segnato goal fuori casa: $Calc\\_ MI\\_ Osp=\pi_{(\\#C)}(\sigma_{(\\#S\ =\ squadra-ospite)}((Marcature\bowtie Calc\\_ MI)\bowtie Partita))$
- Risposta: $(Calc\\_ MI-Calc\\_ MI\\_ Osp)\bowtie Calciatore$
